data/json/items/gun/223.json

```diff
@@ -391,24 +474,108 @@
         "ascii_picture": "m4a1",
         "weight": 1
-      },
+      }
+    ],
+    "to_hit": -1,
+    "bashing": 12,
+    "skill": "rifle",
+    "material": [ "aluminum", "steel", "plastic" ],
+    "symbol": "(",
+    "color": "dark_gray",
+    "ammo": [ "NULL" ],
+    "ranged_damage": { "damage_type": "bullet", "amount": -1 },
+    "dispersion": 180,
+    "durability": 6,
+    "min_cycle_recoil": 1350,
+    "modes": [ [ "DEFAULT", "semi-auto", 1 ], [ "AUTO", "auto", 4 ] ],
+    "default_mods": [ "retool_ar15_223rem_medium" ],
+    "valid_mod_locations": [
+      [ "accessories", 4 ],
+      [ "bore", 1 ],
+      [ "grip", 1 ],
+      [ "mechanism", 4 ],
+      [ "sling", 1 ],
+      [ "stock accessory", 2 ],
+      [ "stock", 1 ]
+    ],
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE_WELL",
+        "item_restriction": [
+          "stanag30",
+          "stanag5",
+          "stanag5ranger",
+          "stanag10",
+          "stanag10ranger",
+          "stanag20",
+          "stanag20ranger",
+          "stanag30ranger",
+          "stanag40",
+          "stanag40ranger",
+          "stanag50",
+          "stanag60",
+          "stanag60drum",
+          "stanag90",
+          "stanag100",
+          "stanag100drum",
+          "stanag150",
+          "stanag20_beowulf",
+          "stanag20ranger_beowulf",
+          "stanag30_beowulf",
+          "stanag30ranger_beowulf",
+          "50beowulf_ar15mag",
+          "450_ar15mag",
+          "762_ar15mag_5rd",
+          "762_ar15mag_10rd",
+          "762_ar15mag_20rd",
+          "762_ar15mag_30rd",
+          "survivor223mag"
+        ]
+      }
+    ]
+  },
+  {
+    "id": "m4_cqbr",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "special duty carbine" },
+    "description": "A rugged military carbine that has been outfitted with various accessories, such as an adjustable buttstock, suppressor, and shortened barrel, to improve its close-quarter battle effectiveness.  Most commonly deployed with special forces units and marines before the Cataclysm, its handy size lends itself well to room clearing and fighting within tight spaces.",
+    "weight": "1316 g",
+    "volume": "1650 ml",
+    "longest_side": "486 mm",
+    "//": "The volume, weight, and overall length of an average AR-15’s complete upper receiver module have been deducted and will be made up for by the installable upper assemblies.",
+    "price": 240000,
+    "price_postapoc": 5000,
+    "variant_type": "gun",
+    "variants": [
       {
         "id": "m4_cqbr",
         "name": { "str": "Mk 18 CQBR" },
-        "description": "This is a shorter M4 carbine with a 10.3 inch barrel, intended for confined spaces and close quarters battle situations."
+        "description": "This is a shorter M4 carbine with a 10.3 inch barrel, intended for confined spaces and close quarters battle situations.",
```

Comment:
I don't think this needs to exist as a seperate item anymore, given it can be assembled from the regular M4

Reply:
> I don't think this needs to exist as a seperate item anymore, given it can be assembled from the regular M4 & a CQB upper.

This is true, but given that gun mods can’t append the name of a weapon, I’d personally feel a bit iffy about up and removing it, as I’d like there to be some form of name difference between a regular M4 and the Mk 18 variant.